src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -70,55 +70,17 @@ public static SafeFreeContextBufferChannelBinding QueryContextChannelBinding(Saf
             throw NotImplemented.ByDesignWithMessage(SR.net_MethodNotImplementedException);
         }
 
-        public static int QueryContextStreamSizes(SafeDeleteContext securityContext, out StreamSizes streamSizes)
+        public static void QueryContextStreamSizes(SafeDeleteContext securityContext, out StreamSizes streamSizes)
         {
             streamSizes = null;
-            try
-            {
-                streamSizes = new StreamSizes(Interop.libssl.SslSizes.HEADER_SIZE, Interop.libssl.SslSizes.TRAILER_SIZE, Interop.libssl.SslSizes.SSL3_RT_MAX_PLAIN_LENGTH);
-                return 0;
-            }
-            catch
-            {              
-                return -1;
-            }
+            streamSizes = new StreamSizes(Interop.libssl.SslSizes.HEADER_SIZE, Interop.libssl.SslSizes.TRAILER_SIZE, Interop.libssl.SslSizes.SSL3_RT_MAX_PLAIN_LENGTH);
         }
 
-        public static int QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
+        public static void QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
         {
             connectionInfo = null;
```

Comment:
My comment (outdated, maybe) was that it was getting set at line 83. Earlier exceptions were handled so the out parameter had to be set.